frontend/conftest.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from botocore.errorfactory import ClientError
+
+from config import Config
+from tests.factories import dynamodb_factory
+
+
+@pytest.fixture
+def dynamodb():
+    return dynamodb_factory(Config.DYNAMODB_ENDPOINT_URL, Config.REGION)
+
+
+@pytest.fixture
+def archive_table(dynamodb):
+    try:
+        table = dynamodb.create_table(
```

Comment:
we can use init script that we already have instead of creating a table in the test. We can assume table exists.